scripts/skia-configuration.ts

```diff
@@ -165,7 +165,6 @@ export const configurations: Configuration = {
     },
     args: [
       ["skia_use_metal", true],
-      ["skia_use_gl", true],
```

Comment:
I think I just assumed that it was superfluous and removed it and it works. Also the skia repository contains a list of build recipes and skia_use_metal=true skia_use_gl=true is not a recipe that exists. Since when this has changed, I don't know.